### DISCUSSION TITLE
How to let longtable caption start from a new line?

### DISCUSSION THREAD

**Discussion by DaftDogeee** [lvjr/tabularray#396] [Q&A]
Hi,

I found some instructions in the manual, but still cannot get the following style.
```latex
\DefTblrTemplate{caption-tag}{default}{TABLE~\thetable}
\DefTblrTemplate{caption-sep}{default}{\par}
\DefTblrTemplate{caption-text}{default}{\InsertTblrText{caption}}
```

![image](https://user-images.githubusercontent.com/49086184/232956209-363aa06b-d0b3-45d7-84a8-5f3b475c4555.png)

**Comment by muzimuzhi** [comment]
The tricky part it that by default the content of `caption` template is typeset and measured in an hbox in which `\par` is ignored.  I recalled there was some example showing the workaround to make the caption content always wider than `\hsize`...
https://github.com/lvjr/tabularray/blob/690e67ee175ff7601d0144c84f9a7e9c67573e0b/tabularray.sty#L4967-L4996

```tex
\documentclass{article}
\usepackage{tabularray}

\DefTblrTemplate{caption-sep}{default}{\par}
\DefTblrTemplate{caption}{default}{%
  \vbox{%
    \UseTblrAlign{caption}%
    \UseTblrTemplate{caption-tag}{default}%
    \UseTblrTemplate{caption-sep}{default}%
    \UseTblrTemplate{caption-text}{default}%
  }%
}

\SetTblrStyle{caption}{c, font=\scshape}

\begin{document}
\begin{longtblr}[caption=Title]{hlines}
  a & a & a & a & a & a \\
  b & b & b & b & b & b
\end{longtblr}
\end{document}
```
![image](https://user-images.githubusercontent.com/6376638/233101335-6af0ab65-02d6-4c10-a9e4-58729a744757.png)

**Reply by lvjr** [reply]
Maybe you are talking about issue #255?

**Reply by DaftDogeee** [reply]
many thanks~
